dev-docs/bidders/opsco.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: bidder
+title: Opsco
+description: Prebid Opsco Bid Adapter
+pbjs: true
+biddercode: opsco
+media_types: banner
+tcfeu_supported: true
```

Comment:
You cannot claim `tcfeu_supported` without a `gvl_id` as it is technically necessary for TCF.

Also please add more meta data. For a full reference see: https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

- safeframes_ok: true/false
- deals_supported: true/false
- floors_supported: true/false
- fpd_supported: true/false
- ortb_blocking_supported
- gpp_sids: tcfeu, tcfca, usnat, usstate_all, usp ( add this field, if you support gpp)

Reply:
@muuki88 Thank you for the review. Changes have been made.

Reply:
@muuki88 Thank you for the review. Changes have been made.